clusterman/migration/worker.py

```diff
@@ -112,14 +117,23 @@ def _drain_node_selection(
     """
     nodes = manager.get_node_metadatas(AWS_RUNNING_STATES)
     selected = sorted(filter(selector, nodes), key=worker_setup.precedence.sort_key)
+    if not selected:
+        return True
+    monitoring_info = {"cluster": manager.cluster, "pool": manager.pool}
+    node_drain_counter = get_monitoring_client().create_counter(SFX_NODE_DRAIN_COUNT, monitoring_info)
+    job_timer = get_monitoring_client().create_timer(SFX_MIGRATION_JOB_DURATION, monitoring_info)
+    node_uptime_gauge = get_monitoring_client().create_gauge(SFX_DRAINED_NODE_UPTIME, monitoring_info)
```

Comment:
What do you think about creating metrics as properties of class (in the constructor) and use it inside method?
similar to autoscaler

Reply:
There's no class is, the worker code is all functional.